src/CompanyService.Models.Db/DbCompany.cs

```diff
@@ -16,18 +18,29 @@ public class DbCompany
         public string Description { get; set; }
         public string Tagline { get; set; }
         public string SiteUrl { get; set; }
+        public string Host { get; set; }
+        public int Port { get; set; }
+        public bool EnableSsl { get; set; }
+        public string Email { get; set; }
+        public string Password { get; set; }
         public DateTime CreatedAt { get; set; }
         public bool IsActive { get; set; }
 
+        [JsonIgnore]
```

Comment:
For what reason were added JsonIgnore?

Reply:
For first record in history i use json convert, and i don't want to have collections in this json